From: Niko Tyni <ntyni@debian.org>
Date: Sun, 25 Sep 2016 09:59:22 +0200
Subject: Temporarily lower the optimization of op.c on hppa due to gcc-6
 problems

This fixes a crash in ext/XS-APItest/t/customop.t

It's currently unclear if the problem is with the code or the
gcc-6 optimizer.

Bug-Debian: https://bugs.debian.org/838613
---
 cflags.SH | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cflags.SH b/cflags.SH
index 443f5fc..bf3a30d 100755
--- a/cflags.SH
+++ b/cflags.SH
@@ -473,6 +473,12 @@ for file do
     # or customize here
 
     case "$file" in
+    op) : work around http://bugs.debian.org/838613
+        case $archname in
+        hppa-*)
+            optimize="$optimize -O0";;
+        esac;;
+
     regcomp) : work around http://bugs.debian.org/754054
         case $archname in
         mips-*|mipsel-*)
